### PR TITLE
Updated Russian translation

### PR DESCRIPTION
**Pull request by IllusiveMan196**
Cleaned up and changed translation file. As per Poedit stats all 194 strings are translated. File resaved in newer version of Poedit.

### CODE REVIEW

File: resources/l10n/ru/LC_MESSAGES/loot.po [hunk 1]
```diff
@@ -26,72 +26,76 @@ msgstr ""
...
+#: src/gui/cef/query/types/sort_plugins_query.h:54
 msgid "Sorting load order..."
 msgstr "Сортировка порядка загрузки..."
 
-#: src/gui/state/loot_state.cpp:95
+#. translators: --auto-sort and --game are command-line arguments and shouldn't be translated.
+#: src/gui/state/loot_state.cpp:106
 msgid "Error: --auto-sort was passed but no --game parameter was provided."
-msgstr "Error: --автосортировка прошла, но --параметр игры не был указан."
+msgstr "Ошибка: --автосортировка прошла, но --параметр игры не был указан."
```
Comment (sibir-ine) [inline resources/l10n/ru/LC_MESSAGES/loot.po:70]: Can you please revert the translations for `--auto-sort` & `--game`? They're command-line arguments & the same across all languages.

Comment (IllusiveMan196) [reply resources/l10n/ru/LC_MESSAGES/loot.po:70]: Use --auto-sort instead of --автосортировка прошла? They were like this in downloaded file

Comment (sibir-ine) [reply resources/l10n/ru/LC_MESSAGES/loot.po:70]: Yeah, that must've been missed when it was initially translated. It'd be best if someone who spoke Russian fixed it as they could tell which translated argument corresponds to which untranslated argument without having to depend on machine translation.

Comment (IllusiveMan196) [reply resources/l10n/ru/LC_MESSAGES/loot.po]: Did it, should look fine now. New pull request for new file?

Comment (RacerBG) [reply resources/l10n/ru/LC_MESSAGES/loot.po]: > 
> 
> Yeah, that must've been missed when it was initially translated. It'd be best if someone who spoke Russian fixed it as they could tell which translated argument corresponds to which untranslated argument without having to depend on machine translation.

--автосортировка -> --auto-sort and --параметр игры -> параметр --game

**Pull request merged**